### PR TITLE
Better UI for long repository description;

### PR DESCRIPTION
**Pull request by Antondomashnev**
These are small bits of UI improvement related to #114 😄 

This is how the long title looks like with changes:

I've changed the margin left & right to `30` and instead of truncating string programatically decided to use `numberOfLines` props of `Text`.

### CODE REVIEW

File: src/components/repository-profile.component.js [hunk 1]
```diff
@@ -123,6 +123,8 @@ export const RepositoryProfile = ({ repository, starred, navigation }: Props) =>
       </Text>
 
       <Text
+        /* @see https://github.com/gitpoint/git-point/issues/114 */
```
Comment (andrewda) [inline src/components/repository-profile.component.js]: Probably don't need this comment

File: src/components/repository-profile.component.js [hunk 2]
```diff
@@ -36,8 +36,8 @@ const styles = StyleSheet.create({
     color: colors.white,
     ...fonts.fontPrimary,
     fontSize: normalize(12),
-    paddingLeft: 15,
-    paddingRight: 15,
+    paddingLeft: 30,
+    paddingRight: 30,
```
Comment (andrewda) [inline src/components/repository-profile.component.js]: _Maybe_ a little bit of top padding?

Comment (junedomingo) [inline src/components/repository-profile.component.js]: for padding left and right, just use `paddingHorizontal: 30`

File: src/components/repository-profile.component.js (hunk 1 above)
Comment (Antondomashnev) [inline src/components/repository-profile.component.js]: I initially wanted to use if we truncate the string, but than found the `numberOfLines` option. I think yeah, we can remove it. Done in 4b55d88

File: src/components/repository-profile.component.js (hunk 2 above)
Comment (Antondomashnev) [inline src/components/repository-profile.component.js]: @junedomingo thanks, didn't know about that before. Used in d2ce96f

> Maybe a little bit of top padding?
For me currently look good, don't you like how the UI looks now?

**Pull request merged**